options/locale/locale_en-US.ini

```diff
@@ -2840,6 +2840,14 @@ settings.location = Location
 settings.permission = Permissions
 settings.repoadminchangeteam = Repository admin can add and remove access for teams
 settings.visibility = Visibility
+settings.change_visibility = Change Visibility
+settings.invalid_visibility = The new visibility is not valid.
+settings.change_visibility_notices_1 = This operation <strong>CANNOT</strong> be undone.
```

Comment:
Are there really any destructive operations in some cases that cannot be undone?

Reply:
Yes, all repository stars will be removed when an organization is converted to private. This behavior follows the logic introduced in [#11717](https://github.com/go-gitea/gitea/issues/11717). Please note that these stars cannot be restored even if the organization is later changed back to public.

Reply:
Maybe this should be somehow more detailed because otherwise it looks like visibility change is final and visibility cannot be changed anymore

Reply:
ecc5b7280041d9809645a192aa99f0afe59af488